x-pack/plugin/ent-search/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
   testImplementation(testArtifact(project(xpackModule('core'))))
   testImplementation project(":test:framework")
   testImplementation(project(':modules:lang-mustache'))
+  testImplementation(project(':modules:reindex'))
```

Comment:
It seems to be needed for `ESSingleNodeTestCase`  to register `Reindex` plugin that exposes `delete_by_query` action